pages/in/widget/pay/index.vue

```diff
@@ -439,40 +440,50 @@ export default {
           paymentRedirectWhiteList,
         };
       });
+      let redirectWhitelistError = null;
+      let isRedirectURLWhiteListed = false;
```

Comment:
Better move these to below, above `if (redirectUri)` block, since that is the part where we handle all redirect related logic